08 - Fun with HTML5 Canvas/index-CLASS.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>HTML5 Canvas</title>
+</head>
+  <body>
+    <canvas id="draw" width="800" height="800"></canvas>
+    <button class="clearCanvas">Clear Canvas</button>
+    <script>
+        //// classes
+        class DrawingCanvas {
+            constructor(id, contextID = '2d') {
```

Comment:
is there any other useful option for `contextId`? if so, then please document it above the constructor.

Reply:
done! I changed the name to `contextType` because this is actually how [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/getContext) defines it:

<img width="300" alt="constructor comments" src="https://user-images.githubusercontent.com/74197346/145840105-effb0e27-4af0-4954-b93c-e035eb5e3b9c.png">

Reply:
ah understand. however, you are making up a new class here. let me ask you this: could you call this class with any other parameter than '2d' and it would still work?
for the `DrawingCanvas` in general this is true. for yours I am not sure. what do you think?

Reply:
on this point... I am aware that the constructor is only set to take a default value '2d' just in case the dev forgets to add the second argument. I just wanted to recap how to pass default values to the constructor.

Furthermore, the class is not set to take any other `contextType`. I would need to set it to take the other Types and to give an error message with any other unaccepted value. 

To set the class properly I would maybe use a switch or if statements based on the `contextType` and set the `DrawingCanvas` consequently with different parameters for each of the 4 types.

Reply:
alrighty. I will merge then :)